server/block/container.go

```diff
@@ -27,3 +28,13 @@ type Container interface {
 	RemoveViewer(v ContainerViewer, w *world.World, pos cube.Pos)
 	Inventory() *inventory.Inventory
 }
+
+// XPContainer represents a block that may contain XP, such as smelters.
+type XPContainer interface {
+	// Experience returns the amount of XP the container has.
+	Experience() int
+	// SetExperience sets the amount of XP the container has.
+	SetExperience(int)
```

Comment:
Are these methods used on this interface? As far as I was able to see, it seemed like `ResetExperience` was the only method used from this interface.

Reply:
No, removed unused ones.